### PR TITLE
Use Bangle's temperature instead of non-working service in advcasio watch

### PR DESCRIPTION
**Pull request by jordimas**
advcasio app uses an external service that has several problems:
* The app lives in an abandoned repository
* The app does not work for serveral reasons, including that the Openweathermap which was using is no longer free

This is a simple fix which uses Bangle's temperature instead (which we know that is not 100% accurate), which is better than the current situation where nothing is ever show.

### CODE REVIEW

File: apps/advcasio/metadata.json [hunk 1]
```diff
@@ -18,8 +18,5 @@
   "storage": [
 			{"name":"advcasio.app.js","url":"app.js"},
 			{"name":"advcasio.img","url":"app-icon.js","evaluate":true}
-  ],
-	"data": [
-			{ "name": "advcasio.data.json", "url": "data.json", "storageFile": true }
```
Comment (bobrippling) [inline apps/advcasio/metadata.json:23]: @gfwilliams is there any way to clean up possible leftover data files on people's watches when they upgrade?

**Review by bobrippling — COMMENTED** [review]
Nice fixes, thanks!

File: apps/advcasio/metadata.json (hunk 1 above)
Comment (gfwilliams) [reply apps/advcasio/metadata.json:23]: Not as far as I'm aware, no. I guess we could add something but in the scheme of things I guess one or two files kicking around isn't the end of the world.

Comment (bobrippling) [reply apps/advcasio/metadata.json:23]: That's fair, thanks!

**Pull request merged**